packages/dso-toolkit/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "dso-toolkit",
   "description": "DSO Toolkit",
   "license": "SEE LICENSE IN DISCLAIMER.txt",
-  "version": "25.0.0",
+  "version": "58.1.0",
```

Comment:
Hiervoor moet dus bij elke release dit nummer goed worden gezet. Of kunnen we dat ook automatiseren?

Reply:
Leuk idee met `genversion`, maar de bron van de versie komt ergens anders vandaan: namelijk uit de build environment. Ik ben er nog niet aan toe om deze versions automatisch te laten updaten. Door gebruik te maken zoals we al elders in het platform doen kunnen we dit nog even voor ons uitschuiven:

* https://github.com/dso-toolkit/dso-toolkit/blob/master/storybook/.storybook/main.ts#L5
* https://github.com/dso-toolkit/dso-toolkit/blob/master/website/docusaurus.config.js#L9

De herbruikbaarheid van deze code is suboptimaal, maar je begrijpt het idee denk ik. Ik begrijp niet zo snel waarom de implementatie van deze twee functies zoveel verschillen

Reply:
In het tweede linkje, wat is `DT_VERSION` en hebben we dat hier nodig?

Reply:
Volgens mij heb ik dat ooit gemaakt om lokaal versies uit te kunnen brengen:

```sh
$Env:DT_VERSION = "1.2.3";
yarn build
yarn build-www
yarn publish # ofzo
```

Ik denk dat we die functionaliteit in het kader van dit issue ook willen.